### PR TITLE
Add DropDown

### PR DESCRIPTION
**Pull request by okrdima**
Add DropDown && fix Header
image
now it looks like this
image
+ i added functionality to DropDown

### CODE REVIEW

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]
fix 'delete' styles

**Comment by okrdima** [comment]
> fix 'delete' styles
I fixed that and added  functionality to DropDown

File: src/components/DropDown/DropDown.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Menu, Dropdown } from 'antd'
+import {
+  DownOutlined,
+  DeleteOutlined,
+  ExclamationOutlined,
+  PushpinOutlined
+} from '@ant-design/icons'
+import { useStoreContext } from '../../context'
```
Comment (GingerBoy14) [inline src/components/DropDown/DropDown.js]: as I know we configure jsconfig for context path

File: src/components/DropDown/DropDown.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      </Menu.Item>
+    </Menu>
+  )
+
+  return (
+    <>
+      <Dropdown
+        overlay={DropDownItem}
+        placement="bottomRight"
+        trigger={['click']}>
```
Comment (GingerBoy14) [inline src/components/DropDown/DropDown.js]: you can type it shorter: trigger="click"

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -13,25 +13,19 @@ const Header = () => {
...
-      <Row wrap={false}>
-        <Col span={12}>
-          <Space direction="vertical" align="start" style={{ width: '100%' }}>
-            <Title level={4}>
-              <Text type="success">{counter} done</Text>
-              <Text strong> / from {store.tasks.length - counter}</Text>
-            </Title>
-          </Space>
+      <Row>
+        <Col flex={'auto'}>
```
Comment (GingerBoy14) [inline src/components/Header/Header.js]: you can use "" instead of {''}

File: src/components/Header/Header.js [hunk 4]
```diff
@@ -13,25 +13,19 @@ const Header = () => {
...
+          <Title level={4}>
+            <Text type="success">{counter} done</Text>
+            <Text strong> / from {store.tasks.length - counter}</Text>
+          </Title>
         </Col>
-        <Col span={12}>
-          <Space direction="vertical" align="end" style={{ width: '100%' }}>
-            <Filter />
-          </Space>
+        <Col flex={'none'}>
```
Comment (GingerBoy14) [inline src/components/Header/Header.js]: the same as previous

**Review by Hierophan7 — APPROVED** [review]

File: src/components/TodoListItem/TodoListItem.js [hunk 5]
```diff
@@ -16,9 +17,11 @@ const TodoListItem = (props) => {
           checked={status.done}
           onClick={() => dispatch({ type: 'SET_DONE', payload: id })}
         />
-        <Text ellipsis={true}>{text}</Text>
+        <Text mark={status.important} ellipsis={true}>
+          {text}
+        </Text>
       </Space>
-      <LeftOutlined onClick={() => console.log('Clicked')} />
+      <DropDown {...props} />
```
Comment (GingerBoy14) [inline src/components/TodoListItem/TodoListItem.js]: DropDown only need **id** of task, why you pushes all props?

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

File: src/components/DropDown/DropDown.js [hunk 6]
```diff
@@ -34,10 +34,7 @@ const DropDown = (props) => {
 
   return (
     <>
```
Comment (GingerBoy14) [inline src/components/DropDown/DropDown.js]: remove unnecessary tag

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**